<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="6" failed="0" total="7" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Functionality Test" duration-ms="31067" started-at="2017-05-23T17:50:28Z" finished-at="2017-05-23T17:50:59Z">
    <groups>
    </groups>
    <test name="Login_Page Test" duration-ms="756" started-at="2017-05-23T17:50:28Z" finished-at="2017-05-23T17:50:28Z">
      <class name="com.testcases.Login_Page">
        <test-method status="PASS" signature="StartTest()[pri:0, instance:com.testcases.Login_Page@1f7030a6]" name="StartTest" is-config="true" duration-ms="4" started-at="2017-05-23T23:20:28Z" finished-at="2017-05-23T23:20:28Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- StartTest -->
        <test-method status="SKIP" signature="testcase1(java.util.Hashtable)[pri:0, instance:com.testcases.Login_Page@1f7030a6]" name="testcase1" duration-ms="8" started-at="2017-05-23T23:20:28Z" data-provider="getData" finished-at="2017-05-23T23:20:28Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Status=, Password=, Runmode=N, Expected_Result=, Email=}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Test case run mode is no]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Test case run mode is no
	at com.testcases.Login_Page.testcase1(Login_Page.java:48)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testcase1 -->
        <test-method status="PASS" signature="endTest()[pri:0, instance:com.testcases.Login_Page@1f7030a6]" name="endTest" is-config="true" duration-ms="5" started-at="2017-05-23T23:20:28Z" finished-at="2017-05-23T23:20:28Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- endTest -->
        <test-method status="SKIP" signature="testcase1(java.util.Hashtable)[pri:0, instance:com.testcases.Login_Page@1f7030a6]" name="testcase1" duration-ms="3" started-at="2017-05-23T23:20:28Z" data-provider="getData" finished-at="2017-05-23T23:20:28Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Status=, Password=hjkhjhdjd^7, Runmode=N, Expected_Result=, Email=jakay}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Test case run mode is no]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Test case run mode is no
	at com.testcases.Login_Page.testcase1(Login_Page.java:48)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testcase1 -->
        <test-method status="PASS" signature="StartTest()[pri:0, instance:com.testcases.Login_Page@1f7030a6]" name="StartTest" is-config="true" duration-ms="0" started-at="2017-05-23T23:20:28Z" finished-at="2017-05-23T23:20:28Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- StartTest -->
        <test-method status="PASS" signature="endTest()[pri:0, instance:com.testcases.Login_Page@1f7030a6]" name="endTest" is-config="true" duration-ms="1" started-at="2017-05-23T23:20:28Z" finished-at="2017-05-23T23:20:28Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- endTest -->
        <test-method status="PASS" signature="StartTest()[pri:0, instance:com.testcases.Login_Page@1f7030a6]" name="StartTest" is-config="true" duration-ms="0" started-at="2017-05-23T23:20:28Z" finished-at="2017-05-23T23:20:28Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- StartTest -->
        <test-method status="SKIP" signature="testcase1(java.util.Hashtable)[pri:0, instance:com.testcases.Login_Page@1f7030a6]" name="testcase1" duration-ms="2" started-at="2017-05-23T23:20:28Z" data-provider="getData" finished-at="2017-05-23T23:20:28Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Status=, Password=545, Runmode=N, Expected_Result=, Email=jakay}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Test case run mode is no]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Test case run mode is no
	at com.testcases.Login_Page.testcase1(Login_Page.java:48)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testcase1 -->
        <test-method status="PASS" signature="StartTest()[pri:0, instance:com.testcases.Login_Page@1f7030a6]" name="StartTest" is-config="true" duration-ms="1" started-at="2017-05-23T23:20:28Z" finished-at="2017-05-23T23:20:28Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- StartTest -->
        <test-method status="PASS" signature="endTest()[pri:0, instance:com.testcases.Login_Page@1f7030a6]" name="endTest" is-config="true" duration-ms="0" started-at="2017-05-23T23:20:28Z" finished-at="2017-05-23T23:20:28Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- endTest -->
        <test-method status="SKIP" signature="testcase1(java.util.Hashtable)[pri:0, instance:com.testcases.Login_Page@1f7030a6]" name="testcase1" duration-ms="2" started-at="2017-05-23T23:20:28Z" data-provider="getData" finished-at="2017-05-23T23:20:28Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Status=, Password=545, Runmode=N, Expected_Result=, Email=}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Test case run mode is no]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Test case run mode is no
	at com.testcases.Login_Page.testcase1(Login_Page.java:48)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testcase1 -->
        <test-method status="PASS" signature="endTest()[pri:0, instance:com.testcases.Login_Page@1f7030a6]" name="endTest" is-config="true" duration-ms="0" started-at="2017-05-23T23:20:28Z" finished-at="2017-05-23T23:20:28Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- endTest -->
        <test-method status="PASS" signature="StartTest()[pri:0, instance:com.testcases.Login_Page@1f7030a6]" name="StartTest" is-config="true" duration-ms="1" started-at="2017-05-23T23:20:28Z" finished-at="2017-05-23T23:20:28Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- StartTest -->
        <test-method status="SKIP" signature="testcase1(java.util.Hashtable)[pri:0, instance:com.testcases.Login_Page@1f7030a6]" name="testcase1" duration-ms="3" started-at="2017-05-23T23:20:28Z" data-provider="getData" finished-at="2017-05-23T23:20:28Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Status=, Password=545, Runmode=N, Expected_Result=, Email=}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Test case run mode is no]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Test case run mode is no
	at com.testcases.Login_Page.testcase1(Login_Page.java:48)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testcase1 -->
        <test-method status="SKIP" signature="testcase1(java.util.Hashtable)[pri:0, instance:com.testcases.Login_Page@1f7030a6]" name="testcase1" duration-ms="2" started-at="2017-05-23T23:20:28Z" data-provider="getData" finished-at="2017-05-23T23:20:28Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Status=, Password=12345678, Runmode=Y, Expected_Result=, Email=jakay34@gmail.com}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Test case run mode is no]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Test case run mode is no
	at com.testcases.Login_Page.testcase1(Login_Page.java:48)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testcase1 -->
        <test-method status="PASS" signature="StartTest()[pri:0, instance:com.testcases.Login_Page@1f7030a6]" name="StartTest" is-config="true" duration-ms="0" started-at="2017-05-23T23:20:28Z" finished-at="2017-05-23T23:20:28Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- StartTest -->
        <test-method status="PASS" signature="endTest()[pri:0, instance:com.testcases.Login_Page@1f7030a6]" name="endTest" is-config="true" duration-ms="0" started-at="2017-05-23T23:20:28Z" finished-at="2017-05-23T23:20:28Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- endTest -->
        <test-method status="PASS" signature="endTest()[pri:0, instance:com.testcases.Login_Page@1f7030a6]" name="endTest" is-config="true" duration-ms="1" started-at="2017-05-23T23:20:28Z" finished-at="2017-05-23T23:20:28Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- endTest -->
      </class> <!-- com.testcases.Login_Page -->
    </test> <!-- Login_Page Test -->
    <test name="Sign_Up Page Test" duration-ms="30277" started-at="2017-05-23T17:50:28Z" finished-at="2017-05-23T17:50:59Z">
      <class name="com.testcases.SignUp_Page">
        <test-method status="PASS" signature="testcase2(java.util.Hashtable)[pri:0, instance:com.testcases.SignUp_Page@528931cf]" name="testcase2" duration-ms="30268" started-at="2017-05-23T23:20:28Z" data-provider="getData" finished-at="2017-05-23T23:20:59Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Status=, Name=Vetri, Mobile=956565635, Password=1234567, Runmode=Y, Expected_Result=, Email=vetrivel@gmail.com}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testcase2 -->
        <test-method status="PASS" signature="StartTest()[pri:0, instance:com.testcases.SignUp_Page@528931cf]" name="StartTest" is-config="true" duration-ms="0" started-at="2017-05-23T23:20:28Z" finished-at="2017-05-23T23:20:28Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- StartTest -->
        <test-method status="PASS" signature="end()[pri:0, instance:com.testcases.SignUp_Page@528931cf]" name="end" is-config="true" duration-ms="0" started-at="2017-05-23T23:20:59Z" finished-at="2017-05-23T23:20:59Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- end -->
      </class> <!-- com.testcases.SignUp_Page -->
      <class name="com.testcases.Login_Page">
        <test-method status="PASS" signature="extentReportStart()[pri:0, instance:com.testcases.Login_Page@1f7030a6]" name="extentReportStart" is-config="true" duration-ms="11" started-at="2017-05-23T23:20:28Z" finished-at="2017-05-23T23:20:28Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- extentReportStart -->
        <test-method status="PASS" signature="extentReportEnd()[pri:0, instance:com.testcases.Login_Page@1f7030a6]" name="extentReportEnd" is-config="true" duration-ms="496" started-at="2017-05-23T23:20:59Z" finished-at="2017-05-23T23:20:59Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- extentReportEnd -->
      </class> <!-- com.testcases.Login_Page -->
    </test> <!-- Sign_Up Page Test -->
  </suite> <!-- Functionality Test -->
</testng-results>
